common/pulp/common/error_codes.py

```diff
@@ -113,6 +113,10 @@
 PLP0043 = Error("PLP0043", _("Database 'write_concern' config can only be 'majority' or 'all'. "
                              "Refer to /etc/pulp/server.conf for proper use."), [])
 PLP0044 = Error("PLP0044", _("The target importer does not support the types from the source"), [])
+PLP0045 = Error("PLP0045", _("The repository cannot be exported because some units have "
```

Comment:
active voice rewrite "have not yet been downloaded" -> "are not yet downloaded" or just "are not downloaded"